docs/audio.rst

```diff
@@ -15,17 +18,42 @@ Functions
 =========
 
 .. py:function:: play(source, wait=True, pin=pin0, return_pin=None)
+                 play(source, wait=True, pin=(pin_speaker, pin=0))
 
     Play the source to completion.
 
-    ``source`` is an iterable, each element of which must be an ``AudioFrame``.
+    ``source`` is an iterable, each element of which must be an ``AudioFrame``
+    or built-in ``Sound``.
 
-    If ``wait`` is ``True``, this function will block until the source is exhausted.
+    If ``wait`` is ``True``, this function will block until the source is
+    exhausted.
 
-    ``pin`` specifies which pin the speaker is connected to.
+    As with the music module, you can use the optional ``pin`` argument to
+    specify the output pin can be used to override the default of
+    ``microbit.pin0``. If you have the latest micro:bit, you can use
+    ``microbit.pin_speaker`` **V2**.
 
-    ``return_pin`` specifies a differential pin to connect to the speaker
-    instead of ground.
+    The pin argument can also take a tuple of two pins, for example
+    ``pin=(pin_speaker, pin=0)`` which would output sound on the built-in
+    speaker and pin 0.
+
+    .. note::
+        Using this argument will disable the default functionality on the
+        **V2** board where the sound is mirrored on speaker and pin0, so
+        ``pin=microbit.pin0`` will only output on the edge connector and
+        ``pin=microbit.pin_speaker`` will only use the built-in speaker.
```

Comment:
To be fair, now that the method signature indicates the default pins are pin_speaker and pin0, this is no longer needed.

Reply:
Removed the note https://github.com/bbcmicrobit/micropython/pull/692/commits/6d088b0e348201f80397f7470e9e83034347beaa